From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 14 Sep 2018 23:14:33 +0200
Subject: shell: Fix list of GIR includes

gjs now relies entirely on introspection data to determine parent
types and implemented interfaces, so in order to have all methods
and properties resolve correctly, we must include the corresponding
GIRs of all types used.

Bug: https://gitlab.gnome.org/GNOME/gnome-shell/issues/574
Forwarded: https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/234
---
 src/meson.build | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index a67a026..2045bab 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -223,7 +223,9 @@ libshell_dep = declare_dependency(link_with: libshell)
 libshell_gir_includes = [
   'Clutter-@0@'.format(mutter_api_version),
   'ClutterX11-@0@'.format(mutter_api_version),
-  'Meta-@0@'.format(mutter_api_version)
+  'Meta-@0@'.format(mutter_api_version),
+  'Gcr-3',
+  'PolkitAgent-1.0'
 ]
 
 if have_networkmanager
